src/ToolServiceProvider.php

```diff
@@ -46,13 +37,10 @@ protected function routes()
                 ->group(__DIR__.'/../routes/api.php');
     }
 
-    /**
-     * Register any application services.
-     *
-     * @return void
-     */
     public function register()
     {
-        //
+        Nova::tools([
+            new StaffBookings(),
+        ]);
```

Comment:
@drewroberts @arwaawan3  - this is an example of tool self registration within its own `ToolServiceProvider` which does not extend from TipoffPackageProvider (its not entirely clear to me whether it should since Nova tools are a bit different than a Laravel package).  Maybe there should be a common base of `TipoffToolProvider`?

Reply:
I'm good with either approach. I just like registering them from the package so it's one less step when using it in an application.

Reply:
Thanks @pdbreen @drewroberts . I will update the tool to register this way 👍